src/Data/HashTable/ST/Basic.hs

```diff
@@ -608,6 +665,10 @@ deletedMarker = 1
 
 
 ------------------------------------------------------------------------------
+trueInt :: Int -> Bool
```

Comment:
I've just realised that neither my question nor my comment afterwards make any sense. (For some reason I thought that  `GHC.Exts` does _not_ export `tagToEnum#`, only `GHC.Prim` does.) Sorry.